lib/internal/Magento/Framework/Interception/Code/InterfaceValidator.php

```diff
@@ -71,11 +70,8 @@ public function validate($pluginClass, $interceptedType)
             $methodType = $this->getMethodType($pluginMethod->getName());
 
             $subject = array_shift($pluginMethodParameters);
-            if (!$this->_argumentsReader->isCompatibleType(
-                $subject['type'],
-                $interceptedType
-            ) || $subject['type'] === null
-            ) {
+            if ($subject['type'] === null
```

Comment:
Wouldn't `\is_null()` be even a bit faster? (to apply everywhere `=== null` is used)

Reply:
Functions are almost always slower than operators

Reply:
That is correct most of the time, but as of PHP 7.0 a list of PHP functions (such as `is_null`, `is_int`, etc) are replaced by opcodes, producing much master code.
See https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3048

Reply:
However I'm not sure it worths it :)